api/src/users/authenticateRoutes.js

```diff
@@ -0,0 +1,33 @@
+import coBody from 'co-body';
+import config from 'config';
+import koa from 'koa';
+import koaRoute from 'koa-route';
+import jwt from 'jsonwebtoken';
+import userRepositoryFactory from './userModel';
+
+const app = koa();
+let userRepository;
+
+app.use(function* init(next) {
```

Comment:
Or/and why not use `this` ? `this.userRepository`? Maybe have a `this.repositories = []` from the server.js, then `this.repositories.user` ?